Skyra.Docker/docker-compose.yml

```diff
@@ -2,23 +2,24 @@ version: "3.7"
 services:
   rabbitmq:
     container_name: rabbitmq
-    image: rabbitmq:management-alpine
+    image: rabbitmq:alpine
```

Comment:
No. Management is good. It exposes a web dashboard where you can among other things

1. send custom messages (for debugging)
2. Releases messages that are stuck in the queue
3. View messages in the queue